BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+# BATS: Bash Automated Testing System
+# This is the BUILD.bazel file for compiling the bats binary.
+# Additional rules might need to be added to enable the other bats libraries.
+
+sh_library(
+    name = "bats_lib",
+    srcs = glob(["libexec/**"]),
```

Comment:
What about `lib/bats-core/*`?

Reply:
Good catch, fixed.  My previous tests didn't have any dependencies there, but it's definitely needed.